src/_cffi_src/openssl/x509v3.py

```diff
@@ -290,7 +299,17 @@
 DIST_POINT_NAME *DIST_POINT_NAME_new(void);
 void DIST_POINT_NAME_free(DIST_POINT_NAME *);
 
+void * X509_get_ext_d2i(const X509 *, int, int *, int *);
 """
 
 CUSTOMIZATIONS = """
+#if OPENSSL_VERSION_NUMBER >= 0x10001000L
```

Comment:
You can obtain the CRLDP of an X509 object (if it exists) using `X509_get_ext`. You can find the index via `X509_get_ext_by_NID` and then use that index in `X509_get_ext`. The resulting output can be cast to a `Cryptography_STACK_OF_DIST_POINT` in cffi, so there's no need to have these functions.